task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -180,9 +180,13 @@ spec:
           TESTPASSED=false
         fi
 
-        OCP_VER_FROM_BASE=$(echo "${BASE_IMAGE}" | sed "s/@.*$//" | sed "s/^.*://")   # strips hash first due to greedy match
-        OCP_VER_MAJOR=$(echo "${OCP_VER_FROM_BASE}" | cut -d '.' -f 1)
-        OCP_VER_MINOR=$(echo "${OCP_VER_FROM_BASE}" | cut -d '.' -f 2)
+        # examines the image tag to derive the target OCP version
+        # assumes this is in the form
+        # image-path:[v]major-digits.minor-digits[@sha...]
+        OCP_VER_FROM_IMG=$(echo "${IMAGE_URL}" | sed -e "s/@.*$//" -e "s/^.*://")   # strips hash first due to greedy match
+        # extracts major digits and filters out any leading alphabetic characters, for e.g. 'v4' --> '4'
```

Comment:
I think we should continue referring to `${BASE_IMAGE}` which is what the author configured in the `FROM` statement of his Dockerfile as for the documentation while I fear that `${IMAGE_URL}` also depends on how he named/configured his application